flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/validation/FlinkResourceValidator.java

```diff
@@ -31,4 +32,13 @@
      * @return Optional error string, should be present iff validation resulted in an error
      */
     Optional<String> validate(FlinkDeployment deployment);
```

Comment:
I think this method should be called `validateDeployment`

Reply:
fixed